api/pkg/s3/datastore/azure/azure.go

```diff
@@ -177,6 +177,26 @@ func (ad *AzureAdapter) DELETE(object *pb.DeleteObjectInput, ctx context.Context
 	}
 	return NoError
 }
+
+func (ad *AzureAdapter) POST(object *pb.RestoreObjectInput, ctx context.Context) S3Error {
+	bucket := ad.backend.BucketName
+	log.Logf("bucket is %v\n", bucket)
+	newObjectKey := object.Bucket + "/" + object.Key
+	blobURL := ad.containerURL.NewBlockBlobURL(newObjectKey)
+	log.Logf("blobURL is %v\n", blobURL)
+	setTierRsp, err := blobURL.SetTier(ctx, azblob.AccessTierCool, azblob.LeaseAccessConditions{})
```

Comment:
Why use AccessTierCool but not TierHot, and what about he aws s3, huawei obs, etc.  We'd better use the same tier.